<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QWidget" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MyResources.qrc">
    <normaloff>:/new/prefix1/MyResources/settings-outline-filled.png</normaloff>:/new/prefix1/MyResources/settings-outline-filled.png</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>511</width>
     <height>351</height>
    </rect>
   </property>
   <widget class="QWidget" name="Sound_Setting">
    <attribute name="title">
     <string>Sound</string>
    </attribute>
    <widget class="QTableView" name="playlistView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>481</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>300</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>471</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="btn_add">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="MyResources.qrc">
            <normaloff>:/new/prefix1/MyResources/document-file-up-upload-icon-19.png</normaloff>:/new/prefix1/MyResources/document-file-up-upload-icon-19.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_stop">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="a"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_previous">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_play">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="MyResources.qrc">
            <normaloff>:/new/prefix1/MyResources/5-512.png</normaloff>:/new/prefix1/MyResources/5-512.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_next">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btn_pause">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="MyResources.qrc">
            <normaloff>:/new/prefix1/MyResources/classica_stop-button_flat-circle-white-on-red_512x512.png</normaloff>:/new/prefix1/MyResources/classica_stop-button_flat-circle-white-on-red_512x512.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="MyResources.qrc"/>
 </resources>
 <connections/>
</ui>
